### DISCUSSION TITLE
filebrowser

### DISCUSSION THREAD

**Discussion by aloispichler** [davatorium/rofi#1728] [Ideas/Feature Requests]
The filebrowser opens the selected file in its assigned program by hitting _ENTER_.
I have expected that _CTRL+ENTER_ (or _CTRL+T_) launches the terminal in the path of the file, but this does not seem to work.
Is there a workflow to achieve this behavior?
Thanks!

**Comment by DaveDavenport** [comment]
No, this is currently not supported.

**Reply by teranex** [reply]
Is this still the case, or was support for something like this added since?

**Reply by DaveDavenport** [reply]
No.
